### ISSUE TITLE
Link to documentation on crates.io does not work

### ISSUE THREAD

**Issue by vks** [BurntSushi/xsv#85]
http://burntsushi.net/rustdoc/xsv/ leads nowhere. https://docs.rs/crate/xsv works.

**Comment by BurntSushi** [comment]
Well, the burntsushi.net link is to https://docs.rs/xsv/0.12.1/xsv/ which in turns seems broken. A bug in docs.rs maybe? Not sure.

**Comment by vks** [comment]
Maybe they changed their web interface? The correct link would be https://docs.rs/crate/xsv/0.12.1

**Comment by BurntSushi** [comment]
I can't really believe they would break every single existing link. For example, this still works: https://docs.rs/fst/0.2.3/fst/

**Comment by vks** [comment]
Good point. The link you are using is what is documented on https://docs.rs/about. This does indeed look like a bug on their end.

I opened https://github.com/onur/docs.rs/issues/133/.

**Comment by vks** [comment]
It seems like this bug is specific to non-library crates, that's why it works for fst but not xsv.
